recipes/cad1/task2/baseline/baseline_utils.py

```diff
@@ -0,0 +1,74 @@
+"""Utility functions for the baseline model."""
+# pylint: disable=import-error
+
+import json
+import logging
+import warnings
+from typing import Tuple
+
+import librosa
+import numpy as np
+import pandas as pd
+from omegaconf import DictConfig
+
+logger = logging.getLogger(__name__)
+
+
+def read_mp3(file_path: str, sample_rate=None) -> Tuple[np.ndarray, int]:
```

Comment:
I think here you can simply be explicit about `sample_rate: Optional[int]=None` asits telling you the type is `int` but the default is `None` which is inconsistent because PEP484 does not allow implied `Optional`, thus we have to just be explicit.